docs/_book.yaml

```
@@ -13,22 +13,74 @@ upper_tabs:
     other:
     - name: Tutorials
       contents:
-      - title: Tutorials overview
-        path: /js/tutorials/
-      - title: Baz tutorial
-        path: /js/tutorials/baz_tutorial
+      - title: Getting Started
+        path: /js/tutorials/index
+      - title: Getting Setup
+        path: /js/tutorials/setup
+
+      - heading: Training Models
+      - title: Fitting a Curve to Two-Dimensional Data
```

```suggestion
      - title: Fit a curve to two-dimensional data
```

If that doesn't fit, try: Fit a curve to 2D data

---

Done